index.html

```diff
@@ -1348,6 +1348,19 @@ <h2>
 MAY be encoded in JSON Web Key (JWK) format using the <code>publicKeyJwk</code>
 property. Both the raw 32-byte base58-btc format and JWK format MUST be
 supported by DID Document Processors. The use of other key formats for RSA MUST
```

Comment:
RSA mentioned on the end of this line too?

Reply:
This was a mistake, will fix.

Reply:
Fixed in 41b0b4781bab8fdfae2cc42990bacc670342e04f.